docs/api-reference/full-text/sorting.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Sorting
+---
+
+## Tiebreaking
+
+By default, search results are ordered in descending order by their BM25 score. If multiple results have identical BM25 scores, the `key_field` is used as a tie-breaker.
+This behavior can be controlled with the `stable_sort` parameter.
+
+If `true`, equally-scored or ordered results will be sub-sorted by their `key_field`. As a
+consequence of score evaluation and sorting, query times will be slower. This is useful for testing or anytime where
+results need to be deterministic but the score itself is not necessary. See the `<index_name>.score_bm25()` function
+for retrieving document scores.
+
+If `false`, scores are not generated and instead results are returned in an un-deterministic index order. The benefit
+of this is that the results are returned as quickly as possible. This is useful for queries that are known to return
+many thousands or millions of rows.
+
+`stable_sort` can be passed into `search`, `score_bm25`, and `snippet`.
+
+```sql
+SELECT * FROM search_idx.search('description:keyboard', stable_sort => true);
+```
+
+## Custom Ordering
+
+By default, the rows are ordered by their BM25 score but we can order them by any [fast field](/api-reference/indexing/fast_fields) instead by specifying `order_by_field`.
```

Comment:
The language here feels different from the rest with the "we can order order", probably because this was written by a user and copy-pasted back.

For consistency with your model of documentation writing, perhaps "By default, the rows are ordered by their BM25 score. They can also be ordered by any [fast field] by specifiying`order_by_field`." or something similar